libethereum/ChainParams.cpp

```diff
@@ -70,6 +70,7 @@ string const c_daoHardforkBlock = "daoHardforkBlock";
 string const c_EIP150ForkBlock = "EIP150ForkBlock";
 string const c_EIP158ForkBlock = "EIP158ForkBlock";
 string const c_byzantiumForkBlock = "byzantiumForkBlock";
+string const c_eWASMForkBlock = "eWASMForkBlock";
```

Comment:
with another fork I think it would be reasonable to make markers in tests. like 
execute this test on  Frontier, Homestead  but not Metropolis. 
and so on. this way we will reduce the amount of tests.

Reply:
I don't fully understand, but some improvements in configuration would be nice. I have some ideas now. We should also discuss this with geth team. They should have more experience.

Also, we can use YAML now for better readability.